.eslintrc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Please add the following rules to your eslint file:

https://eslint.org/docs/latest/rules/newline-before-return
https://eslint.org/docs/latest/rules/no-console
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/newline-after-import.md

Reply:
Done